experiment_manager.py

```diff
@@ -351,18 +365,26 @@ def sweep(self, config):
         outer_variables = Config(*variable_config.param_list[:-1])  # "outer" means all but the inner-most loop
         # N-dimensional loop with itertools.product: # (actually N-1 )
 
+        counter = 1
+        num_jobs = 1
+        for var in outer_variables.param_list:
+            num_jobs=num_jobs*len(var.value)
 
         for indices, vals in enumerated_product(*outer_variables.get_values()):
             # update parameters to current values:
             for i, param in enumerate(outer_variables.param_list):
                 curr_config.set_parameter(name=param.name, value=vals[i])
 
-            print("running...")
+            print(f"running job {counter} out of {num_jobs}...")
             # do 1D sweep on the tracing parameter:
-            print(curr_config.param_list) #TODO only print sweep parameters
+            print('current configuration:')
+            for par in outer_variables.param_list:
```

Comment:
What is `par`?

Reply:
changed to param (it shouldn't conflict with param from the earlier loop right?)